.chloggen/timeout.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack, tempomonolithic
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add unified timeout configuration. It changes the default to 30s.
```

Comment:
Thinking out loud, should we have a single timeout or different timeouts for read and write path?

Reply:
please read the PR description. It has some explanation on this

Reply:
Ah, I jumped directly to the code diff :grimacing: 
Ok, makes sense.